eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -826,3 +826,16 @@ stages:
             useMonoRuntime: true
             targetOS: windows
             targetArchitecture: x86
+
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: Windows_x64_Pgo
```

Comment:
I think the architecture is already appended which is why the other legs don't specify it in their buildName.

Reply:
```suggestion
            buildName: Windows_Pgo
```